webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -261,7 +261,7 @@ export default {
       return this.newsId !== null;
     },
     postDisabled: function () {
-      return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/&nbsp;/g, '').trim();
+      return this.uploading || !this.news.title || !this.news.title.trim() || !this.news.body || !this.news.body.replace(/<p>&nbsp;<\/p>/g, '').replace(/&nbsp;/g, '').trim();
```

Comment:
Could you use https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText to check the content of the ckeditor instance

Reply:
In order to use the innerText property we need to perform it on an element, but if we don't call the news.body inside the computed property then the changes of the article body won't be triggered.